src/analysis/retail/monk/mistweaver/modules/features/Abilities.tsx

```diff
@@ -209,8 +209,8 @@ class Abilities extends CoreAbilities {
         enabled: combatant.hasTalent(TALENTS_MONK.DIFFUSE_MAGIC_TALENT),
       },
       {
-        spell: SPELLS.FORTIFYING_BREW_BRM.id,
-        buffSpellId: SPELLS.FORTIFYING_BREW_BRM_BUFF.id,
+        spell: TALENTS_MONK.FORTIFYING_BREW_TALENT.id,
+        buffSpellId: TALENTS_MONK.FORTIFYING_BREW_TALENT.id,
```

Comment:
in df the talent id is not actually the spell cast id. is that no longer the case?

Reply:
it is, reverting